mkdocs.yml

```diff
@@ -77,3 +77,8 @@ nav:
   - 'Change Log ⏏': CHANGELOG.md
   - 'Contributing': CONTRIBUTING.md
   - 'Code of Conduct': CODE_OF_CONDUCT.md
+
+# Google Analytics. Add export WORKFLOW_GOOGLE_ANALYTICS_KEY="UA-XXXXXXXXX-X" to your ~/.bashrc
```

Comment:
What happens if we haven't done so?

Reply:
I will add an error to the deploys cript.

Reply:
Added.